src/cmd/cmd.cpp

```diff
@@ -442,6 +442,11 @@ int main(int argc, char **argv)
     loop.exec();
 #endif
 
+    if (job->reply()->error() != QNetworkReply::NoError){
+        std::cout<<"Authentication Error\n";
```

Comment:
Actually, that error message is not really accurate. The problem could be anything such as network down or server under maintainnence.